lib/mainloop.c

```diff
@@ -468,7 +468,7 @@ main_loop_init(MainLoop *self, MainLoopOptions *options)
 
   main_loop_init_events(self);
   if (!self->options->syntax_only)
-    control_init(self, resolvedConfigurablePaths.ctlfilename);
+    control_thread_start(self, resolvedConfigurablePaths.ctlfilename);
   setup_signals(self);
```

Comment:
Do we need to be explicit that the control subsystem is in a thread? I think thats an implementation detail.

Reply:
ControlServerLoop?